src/controllers/bet.controller.ts

```diff
@@ -0,0 +1,90 @@
+import { Request, Response } from "express";
+import { AuthenticatedRequestParams } from "../interfaces/authenticatedRequestParams";
+import { ForbiddenActionError } from "../errors";
+import { BetModel } from "../models/bet.model";
+import {
+  betCreationSchema,
+  betUpdateSchema,
+} from "../validation/betValidation.schema";
+
+export const getUserBets = async (
+  req: Request<AuthenticatedRequestParams>,
+  res: Response,
+) => {
+  try {
+    const { connectedUser } = req.params;
+    if (!connectedUser) {
+      throw new ForbiddenActionError();
+    }
+
+    const userBets = await BetModel.find({ userId: connectedUser.userId });
+    res.status(200).json(userBets);
+  } catch (error) {
+    if (error instanceof ForbiddenActionError) {
+      return res.status(403).json({ message: error.message });
+    }
+    res.status(500).send(error);
+  }
+};
+
+export const createBet = async (
+  req: Request<AuthenticatedRequestParams>,
+  res: Response,
+) => {
+  try {
+    const { connectedUser } = req.params;
+    if (!connectedUser) {
+      throw new ForbiddenActionError();
+    }
+
+    const bet = betCreationSchema.parse(req.body);
+
+    const newBet = new BetModel({
+      userId: connectedUser.userId,
+      gameId: bet.gameId,
+      winnerBet: bet.winnerBet,
```

Comment:
Similar to your comment below, there should be a safegard to check if the game has not started yet + if the winner is indeed a playing team in this specific game

Reply:
je le fais dans la prochaine PR :)